src/lib/Scenes/Map/GlobalMap.tsx

```diff
@@ -592,6 +592,43 @@ export const GlobalMapContainer = createFragmentContainer(
         sponsoredContent {
           introText
           artGuideUrl
+          shows(first: 2, sort: START_AT_ASC) {
+            totalCount
+            edges {
+              node {
+                id
+                _id
+                __id
+
+                name
+                status
+                href
+                is_followed
+                exhibition_period
+                cover_image {
+                  url
+                }
+                location {
+                  coordinates {
+                    lat
+                    lng
+                  }
+                }
+                type
+                start_at
+                end_at
+                partner {
+                  ... on Partner {
+                    name
+                    type
+                  }
+                  ... on ExternalPartner {
+                    name
+                  }
+                }
+              }
+            }
+          }
```

Comment:
Not all this data is needed, but I want to keep the data shapes of our shows consistent, because they all get rendered in the same components. This will make it easier to switch them to fragment containers post-launch.